projects/ion/src/lib/select/select.component.html

```diff
@@ -1,4 +1,9 @@
-<div class="ion-select" (click)="handleClick()" data-testid="ion-select">
+<div
+  class="ion-select"
+  (click)="handleClick()"
+  data-testid="ion-select"
+  [class.ion-select__required]="isInvalid()"
```

Comment:
What you think about create a param 'valid' and use here? This is more performative.

Reply:
instead of putting it in a variable I found a more "elegant" way using [HostBinding](https://angular.io/api/core/HostBinding)